governance/proposals/012-set-protocol-fee-cross-bridge.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * This proposal send calls accross the bridge to enable protocol fees
+ * on all supported networks.
+ */
+const { ADDRESS_ZERO, getNetwork } = require('@unlock-protocol/hardhat-helpers')
+const { Unlock } = require('@unlock-protocol/contracts')
+const { IConnext, targetChains } = require('../helpers/bridge')
+const { parseSafeMulticall } = require('../helpers/multisig')
+const { ethers } = require('hardhat')
+const PROTOCOL_FEE_STRING = '0.0001'
```

Comment:
It will be better if the protocol fee is expressed in basis points - as this is how the contract will parse it.